lib/amqp_client.js

```diff
@@ -169,11 +169,11 @@ AMQPClient.prototype.connect = function(url) {
 };
 
 /**
- * Creates a sender link for the given address, with optional link options
+ * Creates a sender link for the given address, with optional link policy
  *
  * @method createSender
- * @param {string} [address]    An address to connect this link to
- * @param {*} [options]         Options used for creating the link
+ * @param {String} [address]    An address to connect this link to
+ * @param {Object} [options]    Policy used for creating the sender link
```

Comment:
yeah I was just going to name it `policy`, but then wondered if it would be confusing having a `policy` and `this.policy`. `linkPolicy` also appealed to me, however, we're already using that in the method itself.

Should we:
  * just call it `policy`
  * call it `senderPolicy`
  * rename `this.policy` => `this.clientPolicy`, call it `policy`
  * <something else>